ballerina/tests/README.md

```diff
@@ -42,7 +37,7 @@ clientSecret = "DUMMY_CLIENT_SECRET"
 
 ### Using Environment Variables
 
-### Linux or macOS
+#### Linux or macOS
```

Comment:
I think we have added default values to the configurable variables and remove the value retrieval from the environment variables. In that case, let's remove this part.